packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -475,8 +469,13 @@ export class VaModal {
       const bodyClass = status ? 'va-modal-alert-body' : 'va-modal-body';
       const titleClass = status ? 'va-modal-alert-title' : 'va-modal-title';
       return (
-        <Host aria-label={ariaLabel} aria-modal="true" role={ariaRole(status)}>
-          <div class={wrapperClass} tabIndex={-1}>
+        <Host>
+          <div class={wrapperClass}
+            role='dialog'
```

Comment:
If we are not toggling between "dialog" and "alertdialog" any longer, is there a utility method we can remove as well?

Reply:
Oh my bad! It is removed. :D